core-web/libs/sdk/experiments/tsconfig.json

```diff
@@ -1,16 +1,24 @@
 {
     "extends": "../../../tsconfig.base.json",
     "compilerOptions": {
-        "module": "commonjs",
-        "forceConsistentCasingInFileNames": true,
+        "target": "ESNext",
+        "useDefineForClassFields": true,
+        "module": "ESNext",
         "strict": true,
-        "noImplicitOverride": true,
-        "noPropertyAccessFromIndexSignature": true,
-        "noImplicitReturns": true,
-        "noFallthroughCasesInSwitch": true
+        "moduleResolution": "Node",
+        "resolveJsonModule": true,
+        "isolatedModules": true,
```

Comment:
Wht this one?

Reply:
interesting one. " Ensures that each file can be transpiled independently without requiring information from other files. "

Reply:
do we need it?